hopbox/hopbox_app/templates/index.html

```diff
@@ -7,19 +7,20 @@
     <title>HopBox Subscription Service</title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
     integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-    <link rel="stylesheet" href=static/style.css>
     {% load static %}
+    <link rel="stylesheet" href=static/style.css>
 </head>
 <body>
     <div id="container">
         <nav class="navbar navbar-expand-lg navbar-white  bg-white mb-3" style="border-bottom: 3px solid #91B957;">
             <div class="container-fluid">
                 <a class="navbar-brand ms-2 fw-bold active" href="/userdashboard">
-                    <img src="{% static 'images/HopBox-logo.png' %}" alt="logo_placeholder" width="340" height="155" class="d-inline-block align-text-top rounded">
+                    {% load static %} <a href="/"> <img class='logo' src="{% static 'images/hopbox-logo-lg.png' %}" alt="HopBox-Logo" /></a>
```

Comment:
Nothing wrong here, I just really like how you nested the image in a link tag so clicking on it takes you to the home page...so cool!